app/assets/v2/js/activity.js

```diff
@@ -739,8 +739,11 @@ $(document).ready(function() {
           <div class="col-sm-1 mt-1 activity-avatar d-none d-sm-inline">
             <img src="/dynamic/avatar/${document.contxt.github_handle}">
           </div>
-          <div class="col-12 col-sm-11 text-right">
+          <div class="comment-area col-12 col-sm-11 text-right">
             <textarea class="form-control bg-lightblue font-caption enter-activity-comment" placeholder="Enter comment" cols="80" rows="3">${existing_text}</textarea>
+            <div class="emoji-container position-absolute d-flex flex-wrap" style="width: 3.6em; right: 0; top: 0">
```

Comment:
could we shift the  inline styles to  `activity_stream.css`
and we can remove `top:0` cause it's already there in top